config/fcos/v1_6_exp/validate.go

```diff
@@ -15,13 +15,26 @@
 package v1_6_exp
 
 import (
+	"regexp"
+
 	"github.com/coreos/butane/config/common"
 	"github.com/coreos/ignition/v2/config/util"
 
 	"github.com/coreos/vcontext/path"
 	"github.com/coreos/vcontext/report"
 )
 
+var regex = regexp.MustCompile(`/(etc|var)`)
+
+func (conf Config) Validate(c path.ContextPath) (r report.Report) {
+	for fs, filesystem := range conf.Storage.Filesystems {
+		if util.NilOrEmpty(filesystem.Path) || (!regex.MatchString(*filesystem.Path)) {
```

Comment:
I just realized we need one more check here: we should only fail if `util.IsTrue(filesystem.WithMountUnit)`.  The semantics of `path` are:

> the mount-point of the filesystem while Ignition is running relative to where the root filesystem will be mounted. This is not necessarily the same as where it should be mounted in the real root, but it is encouraged to make it the same.

So, it's legal for the config to specify arbitrary `path`s for filesystems (which will only be used when Ignition is running) and then manually arrange for the filesystem to be mounted somewhere else.  However, most configs probably set `with_mount_unit: true` to have Butane arrange for the mounting of the filesystem, in which case the `path` does need to be a valid mountpoint.